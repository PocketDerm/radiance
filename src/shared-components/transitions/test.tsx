import React from 'react';
import { renderer } from 'src/tests/reactTestRendererHelpers';

import { FadeInContainer } from './index';

describe('<FadeInContainer />', () => {
  describe('UI snapshot', () => {
    it('renders with default props', () => {
      const component = renderer.create(
        <FadeInContainer>Default Props example</FadeInContainer>,
      );

      const tree = component.toJSON();
      expect(tree).toMatchSnapshot();
    });

    it('renders with slide and speed props set', () => {
      const component = renderer.create(
        <FadeInContainer slide speed="500ms">
          FadeInContainer Content Here
        </FadeInContainer>,
      );

      const tree = component.toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
