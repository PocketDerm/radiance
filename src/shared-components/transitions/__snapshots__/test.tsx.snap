// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FadeInContainer /> UI snapshot renders component and children 1`] = `
@keyframes animation-0 {
  0% {
    -webkit-transform: translateY(-10%);
    -ms-transform: translateY(-10%);
    transform: translateY(-10%);
    opacity: 0;
  }

  25% {
    opacity: 0;
  }

  100% {
    -webkit-transform: translateY(0%);
    -ms-transform: translateY(0%);
    transform: translateY(0%);
    opacity: 1;
  }
}

.emotion-0 {
  -webkit-animation: animation-0 500ms ease-in-out;
  animation: animation-0 500ms ease-in-out;
}

<div
  className="emotion-0 emotion-1"
>
  FadeInContainer Content Here
</div>
`;
